.travis.yml

```diff
@@ -7,8 +7,13 @@ python:
   - "3.5"
 
 node_js:
-  - "0.10"
-
+  - 'node'
 
+env:
+  - TRAVIS_NODE_VERSION="6"
 
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
@evocateur - only on node projects.  This is python project that happens to run node.  Travis would default to node 0.12.  This was fine until this change - webpack requires a newer version of node to work.  Thus incantation.